src/tests/rnp_tests.c

```diff
@@ -171,7 +171,6 @@ main(int argc, char *argv[])
       cmocka_unit_test(test_load_keyring_and_count_pgp),
       cmocka_unit_test(test_load_check_bitfields_and_times),
       cmocka_unit_test(test_load_check_bitfields_and_times_v3),
-      cmocka_unit_test(pgp_compress_roundtrip),
```

Comment:
Do we have some other test which is testing compression? Or not needed anymore?

Reply:
Yeah, CLI tests use compression, so it is tested as well.